azavea/pfb-network-connectivity
# Remove tilemaker

## Overview

Removes all code and config related to the old "tilemaker" static tile pre-rendering process.

### Notes

- I didn't remove the artifacts like batch queues, containers in ECR, and tiles on S3.  The latter is the subject of issue #715.  I don't think the other things really have a marginal cost.
- The `use_tilegarden` variable should be removed from the staging tfvars file when this is merged and from the production file when this is deployed.

## Testing Instructions

- Everything should work the same as before, provided you had `USE_TILEGARDEN` set to true before.
- I ran an analysis job locally to check for regressions and to make sure it went to COMPLETE and showed up on the site.  Running a batch upload seemed like overkill, since there's no reason this should affect that specifically.

## Checklist


Resolves #611
